pkg/cvo/upgradeable.go

```diff
@@ -215,9 +216,26 @@ func (check *clusterVersionOverridesUpgradebale) Check() *configv1.ClusterOperat
 	return cond
 }
 
+type clusterManifestExcludesUpgradable struct {
+	filter payload.RegexpFilter
+}
+
+func (check *clusterManifestExcludesUpgradable) Check() *configv1.ClusterOperatorStatusCondition {
+	if len(check.filter) == 0 {
+		return nil
+	}
+	return &configv1.ClusterOperatorStatusCondition{
+		Type:    configv1.ClusterStatusConditionType("UpgradableManifestExcludes"),
```

Comment:
> https://github.com/openshift/cluster-version-operator/pull/252/files#diff-d9212d08828a9f9d8b54a2e90f31500aR195
the conditions need to start with `Upgradeable`

`UpgradableManifestExcludes` -> `UpgradeableManifestExcludes`

hmm, also this conditions names is not that great.

Reply:
:thinking: